scripts/REFERENCE.md

```
@@ -0,0 +1,142 @@
+  ac-batch --build --qa                  Build all + run QA
+  ac-batch --qa                          QA all built specs
+
+Manage:
+  ac-batch --status                      Show all spec statuses
+  ac-batch --cleanup                     Preview cleanup
+  ac-batch --cleanup --confirm           Delete completed specs
+```
+
+## ac-phase CLI Reference
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)